meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/apalis-imx6/0001-HACK-imx-fiohab-ignore-known-HAB-events.patch

```diff
@@ -0,0 +1,97 @@
+From 92a4ba70c18368943894350327a027f15c5f9589 Mon Sep 17 00:00:00 2001
+From: Michael Scott <mike@foundries.io>
+Date: Thu, 28 Jan 2021 22:06:30 -0800
+Subject: [PATCH] HACK: imx: fiohab: ignore known HAB events
```

Comment:
@vanmaegima @mike-scott can't we make this a bit more generic (still specific to imx6dq) and have it as part of our u-boot-fio tree?